Client.xcodeproj/project.pbxproj

```diff
@@ -391,6 +390,10 @@
 		8A11C8132731E54800AC7318 /* DictionaryExtensionsTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A11C8122731E54800AC7318 /* DictionaryExtensionsTests.swift */; };
 		8A2783F1275FFDC50080D29D /* KeyboardPressesHandler.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A2783F0275FFDC50080D29D /* KeyboardPressesHandler.swift */; };
 		8A2825352760399B00395E66 /* KeyboardPressesHandlerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A2825342760399B00395E66 /* KeyboardPressesHandlerTests.swift */; };
+		8A84D9E1278DED3A0019113B /* PanelDataObservers.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A84D9E0278DED3A0019113B /* PanelDataObservers.swift */; };
+		8A84D9E2278DED3A0019113B /* PanelDataObservers.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A84D9E0278DED3A0019113B /* PanelDataObservers.swift */; };
+		8A84D9E3278DED3A0019113B /* PanelDataObservers.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A84D9E0278DED3A0019113B /* PanelDataObservers.swift */; };
+		8A84D9E4278DED3A0019113B /* PanelDataObservers.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A84D9E0278DED3A0019113B /* PanelDataObservers.swift */; };
```

Comment:
Are these changes just a reorg?

Reply:
I moved the file in this case! It was referenced in Client and in Providers. With some duplication in code as well 🤔